### PR TITLE
 Add output for tests

### PR DESCRIPTION
**Pull request by b0bi79**

### CODE REVIEW

File: ClosedXML.Report.sln [hunk 1]
```diff
@@ -16,40 +16,26 @@ EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
-		Release 3.5|Any CPU = Release 3.5|Any CPU
-		Release 4.5|Any CPU = Release 4.5|Any CPU
-		Release 4.6|Any CPU = Release 4.6|Any CPU
+		Release.Signed|Any CPU = Release.Signed|Any CPU
```
Comment (Pankraty) [inline ClosedXML.Report.sln]: There are many irrelevant changes displayed in this PR. Perhaps you should merge `ClosedXML:develop` into `b0bi79:develop` first.

Comment (b0bi79) [reply ClosedXML.Report.sln]: I already did it, I won't understand why so.

Comment (Pankraty) [reply ClosedXML.Report.sln]: You merged your `master` to your `develop`. You also need merge `upstream:develop` into `origin:develop` and then push it.
image

File: ClosedXML.Report/Excel/Subtotal.cs [hunk 2]
```diff
@@ -257,7 +257,7 @@ private SubtotalGroup CreateGroup(IXLRange groupRng, int groupClmn, int level, s
             }
 
             summRow.Clear(XLClearOptions.Contents | XLClearOptions.DataType); // ClosedXML issue 844
-            summRow.Cell(groupClmn).Value = _getGroupLabel != null ? _getGroupLabel(title) : title + " Итог";
+            summRow.Cell(groupClmn).Value = _getGroupLabel != null ? _getGroupLabel(title) : title + " Total"; // TODO "Total" string move to resources
```
[the remark this answers is not in the document]
Comment (b0bi79) [reply ClosedXML.Report/Excel/Subtotal.cs]: Good idea. Probably and it is necessary to make.

**Pull request merged**